### DISCUSSION TITLE
error when saving mission.

### DISCUSSION THREAD

**Discussion by kk23abc** [Vdauphin/HeartsAndMinds#1140] [Question and answer]
Hello, i have problem saving my mission file. Here's the error message ; 
I hope someone could help guide me how to fix this. Tq

private _fakeViV = isVehicleCargo attachedTo _x;
if (
isNul>
 2:31:28   Error position: <isVehicleCargo attachedTo _x;
if (
isNul>
 2:31:28   Error isvehiclecargo: Type Array, expected Object
 2:31:28 File mpmissions\__cur_mp.WL_Rosche\core\fnc\db\save.sqf..., line 163
 2:31:55 Error in expression <_data pushBack []; 

private _fakeViV = isVehicleCargo attachedTo _x;
if (
isNul>
 2:31:55   Error position: <isVehicleCargo attachedTo _x;
if (
isNul>
 2:31:55   Error isvehiclecargo: Type Array, expected Object
 2:31:55 File mpmissions\__cur_mp.WL_Rosche\core\fnc\db\save.sqf..., line 163

**Comment by LegalSmeagol** [comment]
Having this exact issue. Would appreciate some kind of response or fix.

**Reply by Vdauphin** [reply]
Hello,

I don't know what you guys did. I am not in your computer.
Those user get the same problem but never bother reply to me: https://github.com/Vdauphin/HeartsAndMinds/issues/1117 https://github.com/Vdauphin/HeartsAndMinds/issues/1173

May be you just removed a vehicle in EDEN editor without removing the variable in the `btc_vehicles` or `btc_helo` array

Cheers
